src/app/shared/components/submit-buttons/submit-buttons.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="sumbit-buttons-main">
```

Comment:
it's not worth creating a separate component for those. It will bring redundant complexity. Just use buttons in place